README.md

```diff
@@ -2,101 +2,14 @@
-
-1) Add a user-defined axis in the **Configuration Matrix** to represent the MATLAB versions in the build. Specify the name of the axis in the **Name** field and its values in the **Values** field. Separate the elements in the **Values** field with a space. In this example, four MATLAB versions are specified, which will be used to run the same set of tests.
-
-![image](https://user-images.githubusercontent.com/47204011/62603081-c2c8cc00-b912-11e9-83a4-c5462f58f607.png)
-
-2) In the **Run MATLAB Tests** section of the project, include the user-defined axis name in the **MATLAB root** field to specify the locations where MATLAB is installed. In this example, **$VERSION** will be replaced by one axis value per build step.
-
-![image](https://user-images.githubusercontent.com/51316279/70773424-72b8b500-1d9d-11ea-97ca-807ecd6c4e47.png)
-
-You can select the test artifact check boxes when tests run automatically. Once you have made your selections, save your settings and run the build.
-
-### Option 2: Multi-Configuration Project with Custom Test Mode
-
-To configure the matrix build where you can customize the MATLAB test execution, create a multi-configuration project and follow these steps.
-
-1) Add a user-defined axis in the **Configuration Matrix** to represent the MATLAB versions in the build. 
-
-![image](https://user-images.githubusercontent.com/47204011/62603081-c2c8cc00-b912-11e9-83a4-c5462f58f607.png)
-
-2) Add another user-defined axis using the **Add axis** button. In this example, the **TEST_TAG** axis specifies the possible test tags for a group of test elements.
-
-![image](https://user-images.githubusercontent.com/47204011/62517774-b6c30880-b845-11e9-86a0-8344a281fb27.png)
-
-3) In the **Run MATLAB Tests** section of the project, use the **VERSION** axis to specify the locations where MATLAB is installed.
-
-![image](https://user-images.githubusercontent.com/51316279/70773424-72b8b500-1d9d-11ea-97ca-807ecd6c4e47.png)
-
-4) From the **Test mode** drop-down list, select the **Custom** option. Use the second user-defined axis to create your commands and enter them in the **MATLAB command** field. Then, save your settings and run the build. 
-
-![image](https://user-images.githubusercontent.com/47204011/62686681-cd529680-b9e2-11e9-82c1-c211f1740be4.png)
-
-**Notes:**
-1) For a user-defined axis named **VAR**,  **$VAR** and **${VAR}** are both valid formats for accessing the values.
-
-2) A multi-configuration project creates a separate workspace for each user-defined axis value. If you specify the full paths to where MATLAB is installed as axis values, Jenkins fails to create separate workspaces and fails the build.
+#### Change Log
 
+* For a detailed list of releases, see version history and [change logs](https://github.com/mathworks/jenkins-matlab-plugin/blob/8b77953b23d8a0b0edd40dd2751219f7c3f813b8/CHANGELOG.md). 
```

Comment:
Can't we just say Change logs? 
"For a detailed list of releases, see Change logs."

What exactly is version history in "version history and change logs"?  The phrase implies that "version history" is something comparable to the "change logs" document.

Reply:
Agree Done changed it to For a detailed list of releases, see Change logs.